packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -334,19 +334,29 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
       theme,
       title,
       keytipProps,
-      placeholder,
+      placeholder: placeholderProp,
       tabIndex,
       autofill,
       persistMenu,
-      iconButtonProps
+      iconButtonProps,
+      multiSelect
     } = this.props;
     const { isOpen, focused, suggestedDisplayValue } = this.state;
     this._currentVisibleValue = this._getVisibleValue();
+    const multiselectPlaceholder = multiSelect
+      ? this._getMultiselectDisplayString(this.state.selectedIndices, this.state.currentOptions, suggestedDisplayValue)
+      : '';
 
     const divProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(this.props, divProperties, ['onChange', 'value']);
 
     const hasErrorMessage = errorMessage && errorMessage.length > 0 ? true : false;
 
+    // If the combobox has focus, is multiselect, and has a display string, then use that placeholder
+    // so that the selected items don't appear to vanish. This is not ideal but it's the only reasonable way
+    // to correct the behavior where the input is cleared so the user can type. If a full refactor this
+    // should be removed and the multiselect combobox should behave like a picker.
+    const placeholder = focus && this.props.multiSelect && multiselectPlaceholder ? multiselectPlaceholder : placeholderProp;
```

Comment:
>multiselectPlaceholder [](start = 59, length = 22)

Will this be true if `multiselectPlaceholder` is `''`?

Reply:
No, that evaluates to false.